### ISSUE TITLE
Missing Gitlab report/renderer in latest version.

### ISSUE THREAD

**Issue by koenhendriks** [phpmd/phpmd#977]
- PHPMD version: 2.12.0
- PHP Version: 8.1
- Installation type: composer
- Operating System / Distribution & Version: macOS Monetery 12.4  

According to the documentation on phpmd.org and looking at PR #957 there should be support for gitlab as renderer. 
![Screenshot 2022-08-26 at 11 57 40](https://user-images.githubusercontent.com/4124579/186879392-44c66093-2bbe-40f1-926f-819a9b9a5e7e.png)

### Current Behavior
When latest PHPMD is run it fails to find the gitlab report class with the following error:
`Can't find the custom report class: gitlab`

### Expected Behavior
PHPMD runs successfully and reports a json like output created in #957 

### Steps To Reproduce:

1. Add latest (2.12.0) PHPMD to project with composer: `composer require --dev phpmd/phpmd`
2. Run phpmd with any rule using `gitlab` as reporter: `./vendor/bin/phpmd app gitlab cleancode`
3. Phpmd fails with error that it cant find gitlab report class.

```bash
./vendor/bin/phpmd app gitlab cleancode
Can't find the custom report class: gitlab
```
4. **(optional)** Manually check the `vendor/phpmd/phpmd/src/main/php/PHPMD/Renderer` directory and see that `GitlabRenderer.php` file is missing.

## Checks before submitting
* [x] Be sure that there isn't already an issue about this. See: [Issues list](https://github.com/phpmd/phpmd/issues)
* [x] Be sure that there isn't already a pull request about this. See: [Pull requests](https://github.com/phpmd/phpmd/pulls)
* [x] I have added every step to reproduce the bug.
* [x] If possible I added relevant code examples.
* [x] This issue is about 1 bug and nothing more.
* [x] The issue has a descriptive title. For example:  "JSON rendering failed on Windows for filenames with space".
